content/en/logs/log_collection/javascript.md

```diff
@@ -587,7 +587,18 @@ The placeholders in the examples above are described below:
 
 If your Browser logs contain sensitive information that needs redacting, configure the Browser SDK to scrub sensitive sequences by using the `beforeSend` callback when you initialize the Browser Log Collector.
 
-The `beforeSend` callback function gives you access to each log collected by the Browser SDK before it is sent to Datadog, and lets you update any property.
+The `beforeSend` callback function that you can provide and can be invoked with two arguments: the `log` event and `context`. The function gives you access to each log collected by the Browser SDK before it is sent to Datadog, and lets you use the context to adjust any log properties. The context contains additional information related to the event, but not necessarily included in the event. You can typically use this information to [enrich][18] your event or [discard][19] them.
+
+```javascript
+function beforeSend(log, context)
+```
+
+The potential `context` values are:
+
+| Value | Data Type | Use Case |
+|-------|---------|------------|
+| `isAborted` | Boolean | For network log events, this property tells you whether the failing request was aborted by the application, in which case you might not want to send this event because it might be intentionally. |
```

Comment:
intentionally...? looks like part of the sentence is missing

Reply:
oops! addressed.